airflow/www_rbac/api/experimental/endpoints.py

```diff
@@ -314,15 +348,25 @@ def clear_dag_run(dag_id):
     run_id = data['run_id']
 
     try:
-       dr = _clear_dag_run(dag_id, run_id)
+        with create_session() as session:
+            dr = models.DagRun.find(dag_id=dag_id, run_id=run_id)[0]
+            session.add(models.Log(
+                'api_clear_dagrun',
+                task_instance=None,
+                owner=_get_user_from_header(),
+                dag_id=dag_id,
+                execution_date=dr.execution_date,
+                extra=str(list(data.items()))
```

Comment:
This won't have PII in it right? Only if the payload has PII in it?

Reply:
this is the payload so actually yes it could have PII if someone is calling this endpoint with an arg that has PII